src/ios.ts

```diff
@@ -132,11 +133,23 @@ export const iosApi = <RunTarget extends SupportedRunTarget<'ios'>>(
     target: { platform: 'ios', runTarget: options.runTarget },
     _internal: {
         ssh: async (...args) => {
+            let killProxyProcesss: (() => boolean) | undefined = undefined;
+
+            if (!options.targetOptions?.ip)
+                killProxyProcesss = await startUsbmuxProxy(22161, options.targetOptions?.port || 22);
```

Comment:
You don't need a `let` for _that_…

Reply:
But I do. If use const I cannot reference this outside of the scope of the `if`.

Reply:
```ts
const killProxyProcess = !options.targetOptions?.ip
    ? await startUsbmuxProxy(22161, options.targetOptions?.port || 22)
    : undefined;
```